### ISSUE TITLE
Any plans to upgrade this to latest lib version and Scala 2.12?

### ISSUE THREAD

**Issue by dbuschman7** [cretz/scala-web-ideal#3]

**Comment by cretz** [comment]
Nah, you'll note this even uses old beta akka http libs too. It was more of an experiment to see how I liked the fusion of the different technologies. I'll mention that it is outdated/stale in the description.
